packages/shared/src/components/cards/SourceButton.tsx

```diff
@@ -33,7 +33,7 @@ export default function SourceButton({
           name: source.name,
           image: source.image,
           permalink: getSourcePermalink(source.id),
-          username: source.id,
+          username: source.handle,
```

Comment:
Maybe fallback to id, not sure we set handle for all existing sources?

Reply:
Our DB column for `source.handle` is `NOT NULL` - this should be safe, I think.